beacon-chain/types/state.go

```diff
@@ -48,23 +55,65 @@ func NewGenesisStates() (*ActiveState, *CrystallizedState) {
 		validator := &pb.ValidatorRecord{StartDynasty: 0, EndDynasty: params.DefaultEndDynasty, Balance: params.DefaultBalance, WithdrawalAddress: []byte{}, PublicKey: 0}
 		validators = append(validators, validator)
 	}
+
+	// Bootstrap attester indices for slots, each slot contains an array of attester indices.
+	seed := make([]byte, 0, 32)
+	committee, err := casper.ValidatorsByHeightShard(common.BytesToHash(seed), validators, 1, 0)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	// Starting with 2 cycles (128 slots) with the same committees.
+	committees := append(committee, committee...)
+	// Convert boot strapped attester indices array into proto format.
+	var shardCommittees []*pb.ShardAndCommittee
+	var indexCommittees []uint32
+	for _, committee := range committees {
+		for _, index := range committee.Committee {
+			indexCommittees = append(indexCommittees, uint32(index))
```

Comment:
Why is `indexCommittees` accumulating the previous committee? Shouldn't there be a reset for every iteration? 

Also, is this casting to `uint` to comply with protobuf? Can we keep the type consistent throughout instead?